pages/osx/networkquality.md

```diff
@@ -0,0 +1,20 @@
+# networkQuality
+
+> Measure the network quality by connecting to the internet.
+> More information: <https://support.apple.com/HT212313>.
+
+- Test the network quality for the default interface:
+
+`networkQuality`
+
+- Test sequential upload and download instead of parallel:
```

Comment:
```suggestion
- Tes tthe network quality, by sequentially instead of parallel uploading and downloading:
```

Reply:
Typo, `Tes tthe` → `Test the` @marchersimon?